frontend/src/routes/Symptoms/index.js

```diff
@@ -124,7 +131,7 @@ export default function Symptoms(props) {
       <Fab
         style={{ background: "#9206FF" }}
         aria-label="Go to previous page"
-        onClick={navigate(paths.criticalQuestions)}
+        onClick={navigate()}
```

Comment:
the removal of `paths.criticalQuestion` was intended?

Reply:
Yep! We need to do the generic `history.goBack()` (as seen in `actions/story`) because you can reach the symptoms component from several different places, and when pressing this arrow, you should head back to wherever you were before.